docs/configuration/admin_oidc.rst

```diff
@@ -63,132 +62,126 @@ Detailed Information
 
     Variable            ADMIN_OIDC_CLAIM_MAPPING
     Setting             claim mapping
-    Description         Mapping from user-model fields to OIDC claims
-    Possible values     Mapping: {'some_key': 'Some value'}
-    Default value       {'email': 'email', 'first_name': 'given_name', 'last_name': 'family_name'}
-    
+    Description         Mapping from user-model fields to OIDC claim paths
+    Possible values     Mapping: {'some_key': ['Some value']}
```

Comment:
The possible values are hard-coded for the different field types. Needs change in the library (or overwrite in the configuration step, but this would need to be done for every setting separately)